src/pages/home/HomeBlogPreview.jsx

```diff
@@ -2,26 +2,33 @@ import style from "../../style";
 import { SectionBottom, SectionTop } from "../../layout/Section";
 import { posts } from "../../posts/postTransformers";
 import useDisplayCategory from "../../hooks/useDisplayCategory";
-import Carousel from "../../layout/Carousel";
 import moment from "moment";
-import React, { useState } from "react";
+import React from "react";
 import useCountryId from "../../hooks/useCountryId";
 import EmphasisedLink from "../../layout/EmphasisedLink";
 import { FileImageOutlined } from "@ant-design/icons";
 
 export default function HomeBlogPreview() {
   const countryId = useCountryId();
   const featuredPosts =
-    posts.filter(
+    posts
+      .filter(
+        (post) =>
+          post.tags.includes("featured") &&
+          (post.tags.includes(countryId) || post.tags.includes("global")),
+      )
+      .sort((a, b) => b.date - a.date) || [];
```

Comment:
This is minor, but something I didn't notice before - sort doesn't break on empty arrays
```suggestion
      .sort((a, b) => b.date - a.date);
```